src/Meta.props

```diff
@@ -7,6 +7,6 @@
 <RepositoryUrl>https://github.com/ncthbrt/Fable.Websockets.Elmish</RepositoryUrl>
 <PackageTags>fable;fsharp;elmish;websockets;observables</PackageTags>
 <Authors>Nick Cuthbert</Authors>
-<Version>0.1.1</Version>
+<Version>0.2.0</Version>
```

Comment:
Feel free to change the Meta here to more accurately reflect contributions

Reply:
Thank you. My contribution is very small, and I learned new things from your code. That is the motivation point.